yaroslav.nazarenko/first/task2/task2.rb

```diff
@@ -0,0 +1,13 @@
+hash = {
+    'Yes' => 23,
+    'b' => 'travel',
+    'yesterday' => 34,
+    5 => '234',
+    yesss: :fg,
+    try: 30,
+    key: 'some value',
+    ' yesterday1 '=> 34,
+    ' yesteryear '=> 2014
+}
+numOfkeys = hash.keys.count { |key| key.to_s.downcase.index("yes") == 0 }
+puts "Number of keys: #{ numOfkeys }"
```

Comment:
@desimal911 use snake_case for variables name (and resolve comment after you push the changes)

```suggestion
puts "Number of keys: #{ keys_count }"
```